tests/spec/core/markdown-spec.js

```diff
@@ -284,6 +284,93 @@ describe("Core - Markdown", () => {
     expect(normalBlock.querySelector("code.hljs span")).toBeNull();
   });
 
+  it("is case insensitive for webidl language tags", async () => {
+    const body = `
+      ## test
+
+      \`\`\` weBiDl
+      [Exposed=Window]
+      interface FooWebidl {};
+      \`\`\`
+
+      \`\`\` IdL
+      [Exposed=Window]
+      interface FooIdl {};
+      \`\`\`
+
+      \`\`\` webid
+      [Exposed=Window]
+      interface FooWebid {};
+      \`\`\`
+
+      \`\`\` webidll
+      [Exposed=Window]
+      interface FooWebIdll {};
+      \`\`\`
+
+      \`\`\` idlidl
+      [Exposed=Window]
+      interface FooIdlIdl {};
+      \`\`\`
+
+      \`\`\` webidlwebidl
+      [Exposed=Window]
+      interface FooWebidlWebidl {};
+      \`\`\`
```

Comment:
I would suggest removing these tests, as they seem a bit redundant. 
If the user writes an incorrect language tag, they'll see a `<pre>` without any highlighting - which is enough indication to show they wrote something wrong. Supporting case-insensitivity allows user to not remember/lookup whether it was "WebIDL" or "webIDL" - which is good to have. If they're writing "webID", it's on them :slightly_smiling_face:

Reply:
>which is enough indication to show they wrote something wrong.

The fact that we give enough indication does not mean we don't need a test, does it? 👀

I still think this can somehow be simplified by creating a loop.

Reply:
A loop is a nice middle ground.

Reply:
Hmmmm..... Reading this again, I don't think we need the latter four. (I thought Sid was about the whole test.)